mmdet/core/bbox/transforms.py

```diff
@@ -74,6 +74,14 @@ def bbox2roi(bbox_list):
 
 
 def roi2bbox(rois):
+    """Convert rois to bounding box format
+
+    Args:
+        rois (torch.Tensor): RoIs containing batch id information.
```

Comment:
rois (torch.Tensor): RoIs with the shape (n, 5) where the first column indicates batch id of each RoI.